packages/magic-link/index.js

```diff
@@ -122,3 +128,18 @@ MagicLink.prototype.getUserFromToken = function getUserFromToken(token) {
     });
     return claims.sub;
 };
+
+/**
+ * getLabelsFromToken
+ *
+ * @param {JSONWebToken} token - The token to decode
+ * @returns {object} labels - The labels object associated with the magic link
+ */
+MagicLink.prototype.getLabelsFromToken = function getLabelsFromToken(token) {
```

Comment:
Similar to above, this could just return the claims from the token - otherwise we're gonna have to modify this library everytime we want to attach more/different data to the signup flow

Reply:
Updated 😄 👍